src/core/stdc/assert_.d

```diff
@@ -64,6 +64,10 @@ else version (CRuntime_Glibc)
     ///
     void __assert_perror_fail(int errnum, const(char)* file, uint line, const(char)* func);
 }
+else version (CRuntime_Bionic)
+{
+    void __assert(const(char)* __file, int __line, const(char)* __msg);
```

Comment:
@kinke, need this for the final 1.6 release to build for Android, don't know if ldc itself needs to be patched in any way to call this `__assert` with the right arguments for `betterC`.

Reply:
It does need to be patched as it currently assumes Glibc on Linux.